gyft.py

```diff
@@ -86,20 +30,29 @@
 days[6] = "Saturday"
 #### For day end
 
-timetable_dict = {}
-
-for i in range(1, len(rows)):
-    timetable_dict[days[i]] = {}
-    tds = rows[i].findAll('td')
-    time = 0
-    for a in range(1, len(tds)):
-        if not tds[a].find('b'):
-            continue
-        txt = tds[a].find('b').text.strip()
-        if (len(txt) >= 7):
-            timetable_dict[days[i]][times[time]] = list((tds[a].find('b').text[:7],tds[a].find('b').text[7:], int(tds[a]['colspan'])))
-        time = time + int(tds[a]['colspan'])
-
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", 
+                        help="Input file containing timetable in .txt format")
+    parser.add_argument("-o", "--output",
+                        help="Output file containing timetable in .ics format")
+    parser.add_argument("-d", "--del_events", action="store_true", 
```

Comment:
```suggestion
    parser.add_argument("-d", "--del-events", action="store_true", 
```
Underscore, no one uses that